test/ffmpeg-vaapi/decode/decoder.py

```diff
@@ -27,11 +27,25 @@ def gen_name(self):
       name += "_r2r"
     return name
 
-  def decode(self):
-    self.mformat = mapformat(self.format)
+  def validate_caps(self):
+    if match_best_format(self.format, self.caps["fmts"]) is None:
```

Comment:
test with " ./full run test/ffmpeg-vaapi/decode" , error happen because some decode caps ( avc /vc1) don't have "fmts" keyvalues , please update.

Reply:
What is the error you see?  What is the full command line you used?  The `--platform` command line option is required, so not sure how you got it to run with the command you provided me.

Reply:
ah, yes I see I forgot to finish adding "fmts" key to some of the "libs/caps/" files.  I will update.

Reply:
Fixed.  Please try again.